airbyte-integrations/connectors/source-salesloft/source_salesloft/custom_authenticators.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+#
+
+from dataclasses import dataclass
+from typing import Any, Mapping, Tuple
+
+import requests
+from airbyte_cdk.sources.declarative.auth import DeclarativeOauth2Authenticator
+
+
+@dataclass
+class Oauth2Authenticator(DeclarativeOauth2Authenticator):
```

Comment:
You need use `DeclarativeSingleUseRefreshTokenOauth2Authenticator` as refresh token expired frequently. After that I can run regression tests

Reply:
@lazebnyi I've changed the `OauthAuthenticator` to a `SingleUseOauthAuthenticator` that inherits the `DeclarativeSingleUseRefreshTokenOauth2Authenticator` class.
We're ready to run regression tests!